### DISCUSSION TITLE
How to debug a NotATable exception

### DISCUSSION THREAD

**Discussion by finlaydotb** [delta-io/delta-rs#1325] [Q&A]
I have a remote storage bucket hosted with MinIO with delta table. The problem is when I call it, I get the following error

Error: NotATable("No snapshot or version 0 found, perhaps s3://path/to/table is an empty dir?")

This is the storage options being used

```
    let storage_options_remote = HashMap::from([
        ("allow_http".to_string(), "true".to_string()),
        ("endpoint_url".to_string(), "http://remote.example.com:31090".to_string()),
        ("access_key_id".to_string(), "test@gmail.com".to_string()),
        ("secret_access_key".to_string(), "test.to_string()),
        ("region".to_string(), "eu-central-1".to_string())
    ]);
```

and opening it with

```
    let table = deltalake::open_table_with_storage_options(
        "s3://path/to/table",
        storage_options_remote).await?;
```

Since I can actually navigate to the path, I can confirm that the path does exist and it is not empty...and even more curious, if I download the directory at the path, and upload it to a locally running MinIO, and I change the storage_options to point to locahost:9000, it works!!! I am able to download the data!

So basically same path, same directory downloaded from remote and uploaded to local, but when I query it on remote, I get a reponse of empty table but on local, it works fine.

I am not sure where to start debugging this. Any ideas?

**Comment by finlaydotb** [comment]
While typing this, it occured to me that I might not actually be getting the path right - you know, that part where you get the insight to your problem as you tell someone or write it out. 

Well, I checked and confirmed there was a difference in the base of the paths.
